protocols/7f0f89/7f0f89.ot2.apiv2.py

```diff
@@ -0,0 +1,41 @@
+metadata = {
+    'protocolName': 'Adding Developer Solution to 216 Well Cartridge Plate',
+    'author': 'Rami Farawi <rami.farawi@opentrons.com>',
+    'source': 'Custom Protocol Request',
+    'apiLevel': '2.9'
+}
+
+
+def run(ctx):
+
+    [height_above_cartridge,
+        disp_vol, disp_rate, p300_mount] = get_values(  # noqa: F821
+        "height_above_cartridge", "disp_vol", "disp_rate", "p300_mount")
+
+    if not 0.1 <= height_above_cartridge <= 10:
+        raise Exception("Enter a height between 1 and 10mm")
+    if not 1 <= disp_vol <= 85:
+        raise Exception("Enter a dispense volume between 1 and 85µL")
+
+    # load labware
+    plate = ctx.load_labware('invoy_216_well_cartridge_plate', '1')
+    reservoir = ctx.load_labware('nest_1_reservoir_195ml', '6')
+    tiprack = ctx.load_labware('opentrons_96_tiprack_300ul', '9')
+
+    # load instruments
+    p300 = ctx.load_instrument('p300_single_gen2', p300_mount,
+                               tip_racks=[tiprack])
+
+    # protocol
+    p300.flow_rate.dispense = disp_rate
+    p300.pick_up_tip()
+
+    for i in range(int(len(plate.wells())/3)):
```

Comment:
Another approach here that might be a little cleaner...

1. Create a list of 3-well chunks (or n chunks if you wanted... `n=3`)
`chunks = [plate.wells()[i * n:(i + 1) * n] for i in range((len(plate.wells()) + n - 1) // n )]`

2. Use for-loop with `chunks`
```
for chunk in chunks:
        p300.distribute(disp_vol, reservoir.wells()[0], [well.top(height_above_cartridge) for 
        well in chunk], new_tip='never', blow_out=True, blowout_lcation='source well')

```